### PR TITLE
Issue #3402828: Invalidate cache for ongoing Event

### PR DESCRIPTION
**Pull request by VBaldych**
## Problem
Event with view mode "Featured" still "ongoing" after the event ends.

## Solution
Invalidate cache for ongoing Events.

## Issue tracker
 - https://getopensocial.atlassian.net/browse/PROD-27131
 - https://www.drupal.org/project/social/issues/3402828

## Theme issue tracker
N/A

## How to test
- [ ] Create/edit an event. The end event time should be a few minutes older than the current date. Save
- [ ] Open Layout Builder compatible page (dashboard, for example) in LB
- [ ] Add custom block "Featured content"
- [ ] In the block add your event as a featured content (scroll to the "Items" fieldset)
- [ ] Save LB
- [ ] Wait until the end event time and reload the page - you shouldn't see the "Ongoing" label

### Before merge
- [ ] Code/peer review is completed
- [ ] All commit messages are [clear and clean](https://open-social.slite.com/app/docs/DnmermZDIx_0OQ). If applicable a rebase was performed
- [ ] All automated tests are green
- [ ] Functional/manual tests of the acceptance criteria are approved
- [ ] All acceptance criteria were met
- [ ] New features or changes to existing features are covered by tests, either unit (preferably) or behat
- [ ] Update path is tested. New hook_updates should respect update order, right naming convention and consider hook_post_update code
- [ ] Module can be safely uninstalled. Update/implement hook_uninstall and make sure that removed configuration or dependencies are removed/uninstalled
- [ ] This pull request has all required labels (team/type/priority)
- [ ] This pull request has a milestone
- [ ] This pull request has an assignee (if applicable)
- [ ] Any front end changes are tested on all major browsers
- [ ] New UI elements, or changes on UI elements are approved by the design team
- [ ] New features, or feature changes are approved by the product owner

### After merge
- [ ] Code is tested on all branches that it has been cherry-picked
- [ ] Update hook number might need adjustment, make sure they have the correct order
- [ ] The Drupal.org ticket(s) are updated according to this pull request status

### CODE REVIEW

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

File: modules/social_features/social_event/social_event.module [hunk 1]
```diff
@@ -410,16 +412,24 @@ function social_event_preprocess_node(array &$variables) {
...
+          $render_array = [
+            'enroll_action_form' => $form,
+          ];
+          $variables['event_enrollment'] = $render_array;
+        }
+
+        break;
+
+      case 'featured':
+        $variables['#cache']['max-age'] = 0;
```
Comment (ribel) [inline modules/social_features/social_event/social_event.module:430]: Hi @VBaldych,
It's not a good practice to add max-age 0. 
We need to check if we have other ways to solve this issue.
For example check `social_event_node_view_alter()` where we add some cache tags and context to the even teaser.

Also we need to compare featured view mode with normal teaser.
Srtrange if we have issue only in featured.

**Review by ribel — CHANGES_REQUESTED** [review]

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3612-eiprelklquvjtpmakrhzkwkiyk3fj2tj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/655b50b1382826646cab3a6c

**Review by ribel — APPROVED** [review]
Changes were tested and cache invalidation now works correct (both for Normal teaser and Featured).
Note that ongoing label is not show for anonymous user, but it was the case already before this change and should be tackled in separate issue and PR.


image

image

image

image

**Comment by ribel** [comment]
🍒 picked to the 12.2.x and 12.1.x branch

**Pull request merged**